lib/manager/jenkins/readme.md

```diff
@@ -1,13 +1,4 @@
-The Jenkins manager supports the following format of the plugin list:
+The Jenkins manager supports a custom text or YAML format of the plugin list as described [here](https://github.com/jenkinsci/plugin-installation-manager-tool#plugin-input-format).
+Only versions from the main [update center](https://updates.jenkins.io/) are supported.
```

Comment:
Usually its a good idea to spell out where a link will take you when you click on it.
Also is `update-center` the common name for this registry/website amongst Jenkins users?

```suggestion
Only versions from the main [Jenkins update center](https://updates.jenkins.io/) are supported.
```